addons/website_event/views/event_templates_list.xml

```diff
@@ -117,7 +117,11 @@
         <li class="nav-item dropdown mr-2 my-1">
             <a href="#" role="button" class="btn dropdown-toggle" data-toggle="dropdown">
                 <i class="fa fa-map-marker"/>
-                <t t-if="current_country" t-esc="current_country.name"/>
+                <t t-if="searches['country'] == 'online'">
+                    <t t-set="online_event_text" t-value="'Online Events'"/>
```

Comment:
Won't be translated (independently from being in pot file, it won't even try to translate)
```suggestion
                    <t t-set="online_event_text">Online Events</t>
```